src/renderers/dom/shared/inputValueTracking.js

```diff
@@ -133,9 +133,10 @@ var inputValueTracking = {
       return false;
     }
     var tracker = getTracker(subject);
+    const isNode = (subject: any).nodeType === ELEMENT_NODE;
 
     if (!tracker) {
-      if (typeof (subject: any).tag === 'number') {
+      if (isNode || typeof (subject: any).tag === 'number') {
```

Comment:
O the joy of supporting multiple things at the same time. Ok the lack flow annotations in the event system (and 14 levels of injection and indirection :p) made it difficult to understand what targetInst was type-wise. That's more on me, I'm not super up to date on the fiber internals. 

The check isn't backward I don't think. The problem was node instances were making it to the `track` path instead of the `trackNode` path and exploding. I think the rub that makes this complicated isn't instances vs nodes but the case where a Fiber is passed, it makes the branching really awkward.